controllers/reviews.js

```diff
@@ -0,0 +1,129 @@
+const ErrorResponse = require("../utils/errorResponse");
+const asyncHandler = require("../middleware/async");
+const Review = require("../models/Review");
+const Category = require("../models/Category");
+
+// @desc    GET  reviews
+// @route   GET/api/v1/reviews
+// @route   GET/api/v1/categorys/:categoryId/reviews
+// @access  Public
+
+exports.getReviews = asyncHandler(async (req, res, next) => {
+  // check if the category exist
+  if (req.params.categoryId) {
+    const reviews = await Review.find({ category: req.params.categoryId });
+
+    return res.status(200).json({
```

Comment:
Doing `if/else` and sending the response in each of the blocks is anti-code. Please refactor. Since in your else statement the params `categoryId` does not exists.